enable/null/oldagg.py

```diff
@@ -0,0 +1,18 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+from .image import (
```

Comment:
I believe this is similar to Rahul's question, but why do we pull from `.image` not `.agg` in these?

Because if we eventually have image point to celiagg, then oldagg effectively becomes an alias for celiagg not agg?

Reply:
> `.agg` doesn't exist.

I am still a little confused.  how does the code in `kiva/image.py` work then as it seems to just import from `.agg`, see: https://github.com/enthought/enable/blob/fe462b81d6b815b1ef5b673e80d9c956d87f0b39/kiva/image.py#L19

My understanding was that image was basically just an alias pointing at agg, and it would move to point to celiagg in the future (ie those imports change to `.celiagg`). 

If the image contents are going to eventually be moved into these oldagg modules when `.image` points to celiagg that makes sense.  But the image contents seem to be pulling from `.agg` so it would make sense to me if these files effectively just copied that code over now rather than later?

I believe I am still missing something

Reply:
ahhh I see, my mistake, that makes sense now
thank you!